tests/conftest.py

```diff
@@ -1,8 +1,42 @@
 import pytest
+from genologics_mock.lims import MockLims
+from genologics_mock.entities import (
+    MockArtifact,
+    MockProcess,
+    MockProcessType,
+    MockSample,
+)
+from .helpers import Helpers
 
 
 @pytest.fixture
 def config():
     """Get file path to invalid csv"""
 
-    return "tests/fixtures/config.yaml"
\ No newline at end of file
+    return "tests/fixtures/config.yaml"
+
+
+@pytest.fixture
+def lims():
+    return MockLims()
+
```

Comment:
Regarding the other suggestion, could could look like this:
```suggestion

@pytest.yield_fixture
def lims():
    yield MockLims()

```